lib/personalAccessKey.ts

```diff
@@ -162,22 +179,16 @@ export async function accessTokenForPersonalAccessKey(
 export async function enabledFeaturesForPersonalAccessKey(
   accountId: number
 ): Promise<{ [key: string]: number } | undefined> {
-  const account = getAccountConfig(accountId) as PersonalAccessKeyAccount;
-  if (!account) {
-    throwErrorWithMessage(`${i18nKey}.errors.accountNotFound`, { accountId });
-  }
-  const { auth, personalAccessKey, env } = account;
-  const authTokenInfo = auth && auth.tokenInfo;
-
-  const accessTokenResponse = await getNewAccessToken(
-    accountId,
-    personalAccessKey,
-    authTokenInfo && authTokenInfo.expiresAt,
-    env
-  );
+  const accessTokenResponse = await getAccessTokenResponse(accountId);
   return accessTokenResponse?.enabledFeatures;
 }
 
+export async function scopeGroupsForPersonalAccessKey(
```

Comment:
Nit: We call it a `AccessToken` in `getAccessTokenResponse` and `AccessKey` here, could we standardize

Reply:
@wung-hubspot, did you address this?